pkgs/applications/terminal-emulators/mlterm/default.nix

```diff
@@ -62,20 +165,13 @@ stdenv.mkDerivation rec {
   '';
 
   configureFlags = [
-    "--with-imagelib=gdk-pixbuf" #or mlimgloader depending on your bugs of choice
```

Comment:
Where did the imagelib flag go?

Reply:
> Where did the imagelib flag go?

Oh I tried to debug this for way two long. Here's the story:

Upstream sort of intended that it'd be possible to configure the library that `mlimgloader` tool is using. However, have a look at the following excerpt from the `./configure` script:

https://github.com/arakiken/mlterm/blob/3dfd3b7192a8a132c568a76ad5bcd246eebdc555/configure#L17839-L17844

Upstream decided at some point to discard support for `imlib{,2}` - the only other options that are accepted by the script. In addition, here on NixOS (from some reason) we don't have this `imlib2-config` executable available. I tried to workout a patch that will replace `imlib2-config` with `pkg-config`, but no only upstream wouldn't support this, apparently both `gtk3` and `vte` propagate the `gdk-pixbuf` input anyway - so using `imlib2` has no benefit for a Nix user what so ever.

Also note, that apparently this flag is not even considered by `./configure` when you also use `--without-x`. So to reduce complexity that has no benefit, I decided to remove this flag altogether. Fixing https://github.com/arakiken/mlterm/issues/69 would also help reduce the confusion regarding all the different features, which is also possible via a rather simple `./configure` patch, but I don't want to change upstream's behavior in the meantime - we can do it after we get a reply from them.